content/advanced/340_appmesh_flagger/conclusion.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Conclusion"
+date: 2021-03-18T00:00:00-05:00
+weight: 60
+draft: false
+---
+
+Congrats on automating the rolling out new version of `detail` service using Flagger in AWS App Mesh!
```

Comment:
Perhaps we can say

Congratulations on using AWS App Mesh's Flagger to automate the deployment of a new version of the `detail` service!

Reply:
Changed to this "Congratulations on using Flagger in AWS App Mesh to automate the deployment of a new version of the `detail` service!" as Flagger is not App Mesh's component.